README.md

```diff
@@ -16,64 +16,66 @@
   </a>
 </p>
 
+## Table of Contents
+
 - [Wiki](https://github.com/gojek/ziggurat/wiki)
 - [Release Notes](https://github.com/gojek/ziggurat/wiki/Release-Notes)
 - [Upgrade Guide](https://github.com/gojek/ziggurat/wiki/Upgrade-guide)
-
----
-
 - [Description](#description)
 - [Dev Setup](#dev-setup)
 - [Usage](#usage)
-- [Configuration](#configuration)
+- [Configuration](doc/configuration.md)
```

Comment:
Add concepts and other readme also here in content

Reply:
Its added just below this in important concepts. 
This table of contents is just for this page .